pkl-tools/pkl-tools.gradle.kts

```diff
@@ -51,15 +51,16 @@ dependencies {
 }
 
 // TODO: need to figure out how to properly generate javadoc here.
-// For now, we'll include a dummy javadoc jar.
-val javadocDummy by tasks.registering(Javadoc::class) { source = dummy.allJava }
+// For now, we'll include a placeholder javadoc jar.
+val javadocPlaceholder by tasks.registering(Javadoc::class) { source = placeholder.allJava }
 
 java { withJavadocJar() }
 
 val javadocJar by
   tasks.existing(Jar::class) {
-    from(javadocDummy.get().outputs.files)
+    from(javadocPlaceholder)
     archiveBaseName.set("pkl-tools-all")
+    archiveClassifier.set("javadoc")
```

Comment:
Note that you don't need to set the classifier explicitly as `withJavadocJar()` will create the javadocJar task with the correct classifier. No need to fix it, just wanted to explain it for clarity :)